.github/workflows/run-onecc-build.yml

```diff
@@ -0,0 +1,98 @@
+name: Run ONECC Ubuntu Build
+
+on:
+  pull_request:
+    branches:
+      - master
+      - release/*
+    paths:
+      - '.github/workflows/run-onecc-build.yml'
+      - 'nnas'
+      - 'nncc'
+      - 'compiler/**'
+      - 'infra/cmake/**'
+      - 'infra/nncc/**'
+      - 'nnpackage/schema/**'
+      - 'res/**'
+      - '!**/*.md'
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  onecc-test:
+    runs-on: ubuntu-22.04
```

Comment:
this is jammy be there are `-focal` image used.
this may give some confusion for readers. maybe add some comments about this?

Reply:
OK. I'll change runner to `ubuntu-latest` and add comment about image.